Makefile

```diff
@@ -601,13 +601,18 @@ fmt: ## Format the code using Ruff
 	python -m ruff check --fix-only
 	python -m ruff format
 
+.PHONY: fmt-docs
+fmt-docs: ## Format the code blocks in markdown files
+	@echo "Formatting the docs"
+	git ls-files -z -- '*.md' | xargs -0 blacken-docs -t=py39
+
 .PHONY: lint-imports
 lint-imports: ## Validates import dependencies
 	@echo "Running import linter"
 	lint-imports
 
 .PHONY: lint
-lint: lint-check lint-imports ## Run lint on the code
+lint: lint-check lint-imports fmt-docs ## Run lint on the code
```

Comment:
you should run `fmt-docs` in `make fmt` and not in `make lint`
and in `make lint` you should lint the code with `blacken-docs` using the `--check` flag

Reply:
> you should run `fmt-docs` in `make fmt` and not in `make lint`

I thought it's better to separate `fmt-docs` from `fmt` that runs on actual code, but I'll change it :)

> and in `make lint` you should lint the code with `blacken-docs` using the `--check` flag

I wanted to use it - but the package is yet to be released with this relatively new flag.
I opened https://github.com/adamchainz/blacken-docs/issues/346 to request a new release.

Reply:
you can separate `make fmt-docs` from `make fmt` but I don't think it should be in `make lint`

Reply:
I changed it now to be ready for the next release, but I kept it in `make lint` as only that is enforced in the CI